src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -275,10 +275,19 @@ public enum OrganelleGroup
     ///   The model info returned like this (as it may be a struct type this can't return a nullable reference without
     ///   boxing)
     /// </para>
+    /// <para cref="upgrades">
```

Comment:
```suggestion
    /// <param cref="upgrades">
```

Typo in the XML tag.

Reply:
When I do as you suggested, my IDE complains that it is lacking a name. I tried replacing cref= with name=, it stopped complaining but nothing really changed when I hovered my mouse above the function.

Reply:
The right syntax is:
```c#
/// <param name="useDefaults">If true default checks are added automatically</param>
```
(just a random example I grabbed from Thrive).